SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialTraceEventDataBuilder.h

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "Interop/Connection/SpatialTraceUniqueId.h"
+#include "SpatialCommonTypes.h"
+#include <WorkerSDK/improbable/c_trace.h>
+#include <WorkerSDK/improbable/c_worker.h>
+
+#define GDK_EVENT_NAMESPACE "unreal_gdk."
+
+DECLARE_LOG_CATEGORY_EXTERN(FSpatialTraceEventDataBuilderLog, Log, All);
+
+namespace worker
+{
+namespace c
+{
+struct Trace_EventData;
+}
+} // namespace worker
+
+namespace SpatialGDK
+{
+struct SPATIALGDK_API FSpatialTraceEventName
+{
+	static const char* PushRPCEventName;
```

Comment:
Nit: 
Maybe a #define makes more sense here? For two reasons
- No symbol names are produced (or linking required)
- Used in a single place so the compiler can choose the most appropriate place to put the data (instead of all together in this cpp)

Reply:
Or simply just the string used in place, if there isn't a benefit to keeping them together

Reply:
Some are used multiple times so I am quite happy to keep them in one spot. Also might be a decent reference to see all native gdk events.